### PR TITLE
Fix in import of common.py

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

File: RAMP/module_metadata.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import hashlib
 import platform
 from typing import List, Dict, Any  # noqa: F401
+from .common import eprint
```
Comment (swilly22) [inline RAMP/module_metadata.py:5]: Why do we import `eprint` if we're not using it?

File: RAMP/commands_discovery.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import os.path
 from RAMP.disposableredis import DisposableRedis
+from .common import eprint
```
Comment (swilly22) [inline RAMP/commands_discovery.py:3]: Why do we import `eprint` if we're not using it?

File: RAMP/unpacker.py [hunk 3]
```diff
@@ -2,6 +2,8 @@
 import re
 from zipfile import ZipFile, BadZipfile
 from typing import Dict, Any, IO, Tuple, Optional  # noqa: F401
+from .common import eprint
```
Comment (swilly22) [inline RAMP/unpacker.py:5]: Why do we import `eprint` if we're not using it?

**Comment by rafie** [comment]
As common.py is a place for easily adding common (general/utility) functionality, we might be better off just going `from .common import *`?

**Comment by swilly22** [comment]
@rafie, I don't have a strong opinion on the matter,
I feel like each file should bring in whatever it is using, but I don't mind going with your suggestion, let me know what do you prefer.

**Comment by rafie** [comment]
@swilly22 I prefer the import * for the internal common module, explicit imports for other modules. I'll push the update.

**Comment by swilly22** [comment]
OK, waiting for that update.

**Pull request merged**